parser/ParseStatement.hpp

```diff
@@ -885,6 +885,73 @@ class ParseStatementQuit : public ParseStatement {
   DISALLOW_COPY_AND_ASSIGN(ParseStatementQuit);
 };
 
+/**
+ * @brief Class to hold the parsed command name and an optional argument string.
+ * @details Commands are non-sql statements which can be issued to quickstep.
+ *          They are entered into the CLI as '.command-name command-string\n'.
+ */
+class ParseCommand : public ParseStatement {
+ public:
+  ParseCommand(const int line_number,
+               const int column_number,
+               ParseString *command,
+               ParseString *arguments)
```

Comment:
I note that the rest of the line after the command is passed in as a single string. In this way the splitting of arguments is left to the execution logic. I think it might be better to let lexer do the splitting, the same as command-line arguments which are separated by whitespaces and passed in as a vector of strings.